lms/tasks/celery.py

```diff
@@ -38,6 +38,12 @@
         "interval_start": 0.2,
         "interval_step": 0.2,
     },
+    # Automatically try to establish the connection to the AMQP broker on Celery startup if it is unavailable.
+    broker_connection_retry_on_startup=True,
```

Comment:
Taking the oportinty to fix this warning in the logs, see:

fba2c548526362f08c662dfdb718a08fd0a2ba7d

https://docs.celeryq.dev/en/stable/userguide/configuration.html#broker-connection-retry

https://docs.celeryq.dev/en/stable/userguide/configuration.html#broker-connection-max-retries

Reply:
Nice!